OutOfSchool/OutOfSchool.WebApi/Services/AreaAdminService.cs

```diff
@@ -166,6 +166,7 @@ public async Task<SearchResult<AreaAdminDto>> GetByFilter(AreaAdminFilter filter
         {
             var regionAdminDto = await regionAdminService.GetByUserId(currentUserService.UserId).ConfigureAwait(false);
             filter.InstitutionId = regionAdminDto.InstitutionId;
+            filter.CATOTTGId = regionAdminDto.CATOTTGId;
```

Comment:
(Logic) You've fixed the bug, definitely, but I'm wondering if other bug is not reproduced? 
Consider the case when `GetByFilter()` is called with `filter.CATOTTGId` already set (it might be child of `regionAdminDto.CATOTTGId`) then your fix will return all admins from the region, instead of that one smaller OTG.

Reply:
Since this method is called without the ability to explicitly specify the CATOTTGId for the region admin, I thought it was okay to leave it that way.
But i did not think that this can be changed in the future. I will rework it, to work properly, when `filter.CATOTTGId` is not empty.